src/System.Management.Automation/help/HelpCommands.cs

```diff
@@ -647,6 +666,43 @@ private void LaunchOnlineHelp(Uri uriToLaunch)
             }
         }
 
+#if !UNIX
+        private bool WebBrowserAvailable()
+        {
+            RegistryKey regKey = null;
+            try
+            {
+                // Query the registry and remove the quotes '"' from the path.
+                regKey = Registry.ClassesRoot.OpenSubKey("HTTP\\shell\\open\\command", false);
+                string browserPath = regKey.GetValue(null).ToString().Replace((char)34 + "", "");
+                if (browserPath != null)
+                {
+                    if (!browserPath.EndsWith(".exe", StringComparison.OrdinalIgnoreCase))
+                    {
+                        // Remove any extra chars in the path after ".exe".
+                        browserPath = browserPath.Substring(0, browserPath.LastIndexOf(".exe") + 4);
+                    }
+
+                    // Make sure the path to the default browser exists.
+                    return (File.Exists(browserPath));
+                }
+            }
+            catch (Exception)
+            {
+                // ignore the error. By default we return false.
+            }
+            finally
+            {
+                if (regKey != null)
+                {
+                    regKey.Dispose();
```

Comment:
You can replace this by `using (RegistryKey regKey = ...) { ... }` syntax.
It's more modern way of doing the same.